### ISSUE TITLE
jquery issue and security patch

### ISSUE THREAD

**Issue by jthomasmock** [rstudio/distill#132]
Hi all,

I am getting GitHub-level warnings about jquery (suggesting upgrade to >= 3.5), whereas `distill` has a dependency of 1.1.

This is also leading to Netlify throwing warnings (security issue) and I believe this is also leading to Twitter blocking some links to `distill` with an "Unsafe Link" Warning.

**Comment by jjallaire** [comment]
The jquery dependency comes from the version embedded in rmarkdown. @yihui Could we update the jquery dependency there?

**Comment by yihui** [comment]
Yes, we will upgrade jQuery: https://github.com/rstudio/rmarkdown/issues/1814 Thanks for the report!
